tutor/src/components/buttons/teacher-become-student.js

```diff
@@ -17,6 +18,7 @@ const BecomeButton = styled(Button).attrs({
     border-bottom-width: 1px;
   }
 `;
+BecomeButton.displayName = 'BecomeButton';
```

Comment:
yeah setting the `displayName` is helpful for debugging.  Sometimes when the component has too many decorators applied the displayName is lost so that it looks like just `Component` or something like that to `jest` instead of a properly named one.